code/messaging/src/phone/java/com/adobe/marketing/mobile/MessagingInternal.java

```diff
@@ -537,4 +620,9 @@ private boolean isSharedStateUpdateFor(final String stateOwnerName, final Event
 
         return stateOwnerName.equals(stateOwner);
     }
+
+    // for unit tests
+    protected MessagingCacheUtilities getMessagingCaching() {
```

Comment:
Let's make it default type instead of protected. Event better lets make the variable `messagingCacheUtilities` default type.

Reply:
i've removed this unit test helper function as it wasn't being used.